Carmu/Sources/Views/CrewMake/InviteCodeInputView.swift

```diff
@@ -0,0 +1,183 @@
+        ]
+        friendSearchTextField.attributedPlaceholder = NSAttributedString(
+            string: "초대코드를 입력하세요",
+            attributes: placeholderAttributes
+        )
+
+        return friendSearchTextField
+    }()
+
+    // MARK: - 텍스트 필드 clear 버튼
+    lazy var clearButton: UIButton = {
+        let clearButton = UIButton(type: .custom)
+        clearButton.setImage(UIImage(systemName: "xmark.circle.fill"), for: .normal)
+        clearButton.tintColor = UIColor.semantic.textBody
+        return clearButton
+    }()
+
+    lazy var conformCodeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "확인되었습니다. 즐거운 카풀 여정되세요!"
+        label.font = UIFont.carmuFont.body1
+        label.textColor = UIColor.semantic.textTeriary
+        label.isHidden = true
+        return label
+    }()
+
+    lazy var rejectCodeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "초대코드가 잘못되었어요. 확인 후 다시 입력해주세요."
+        label.font = UIFont.carmuFont.body1
+        label.textColor = UIColor.semantic.textNewCrew
+        label.isHidden = true
+        return label
+    }()
+
+    lazy var nextButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("여정 알리기", for: .normal)
+        button.backgroundColor = UIColor.semantic.accPrimary
+        button.titleLabel?.font = UIFont.carmuFont.headline2
+        button.setTitleColor(UIColor.semantic.textSecondary, for: .normal)
+        button.setBackgroundImage(
+            UIImage(color: UIColor.semantic.textSecondary ?? .white),
+            for: .highlighted
+        )
+        button.layer.cornerRadius = 30
+        return button
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override func draw(_ rect: CGRect) {
+        setupViews()
+        setAutoLayout()
+    }
+
+    private func setupViews() {
+        firstLineTitleStack.axis = .horizontal
+        firstLineTitleStack.alignment = .center
+        secondLineTitleStack.axis = .horizontal
+        secondLineTitleStack.alignment = .center
+
+        firstLineTitleStack.addArrangedSubview(titleLabel1)
+        firstLineTitleStack.addArrangedSubview(titleLabel2)
+        firstLineTitleStack.addArrangedSubview(titleLabel3)
+
+        secondLineTitleStack.addArrangedSubview(titleLabel4)
+        secondLineTitleStack.addArrangedSubview(titleLabel5)
+
+        codeSearchTextFieldView.addSubview(codeSearchTextField)
+        codeSearchTextFieldView.addSubview(clearButton)
+
+        addSubview(firstLineTitleStack)
+        addSubview(secondLineTitleStack)
+        addSubview(codeSearchTextFieldView)
+        addSubview(conformCodeLabel)
+        addSubview(rejectCodeLabel)
+        addSubview(nextButton)
+    }
+
+    private func setAutoLayout() {
+        firstLineTitleStack.snp.makeConstraints { make in
+            make.top.equalTo(safeAreaLayoutGuide.snp.top).offset(78)
```

Comment:
```suggestion
            make.top.equalTo(safeAreaLayoutGuide).offset(78)
```
safeAreaLayoutGuide도 암시가 적용되는 걸로 알고 있어서 위 코드대로 적으면 더 짧아질 것 같습니다!